agent/cache/request.go

```diff
@@ -48,4 +48,17 @@ type RequestInfo struct {
 	// if there was no prior value). This "last known value" behavior matches
 	// normal Consul blocking queries.
 	Timeout time.Duration
+
+	// MaxAge if set limits how stale a cache entry can be. If it is non-zero and
+	// there is an entry in cache that is older than specified, it is treated as a
+	// cache miss and re-fetched. It only really makes sense for cache types that
+	// have Refresh: false.
```

Comment:
"It only really makes sense" is unclear to me. Does it do anything otherwise? etc. Docs note.